### ISSUE TITLE
4 New submissions for Mon, 29 Jan 24

### ISSUE THREAD

**Issue by PanagiotisFytas** [PanagiotisFytas/get-daily-arxiv-noti#453]
### Additional Look into GAN-based Augmentation for Deep Learning COVID-19  Image Classification
 - **Authors:** Oleksandr Fedoruk, Konrad Klimaszewski, Aleksander Ogonowski, Michał Kruk
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2401.14705
 - **Pdf link:** https://arxiv.org/pdf/2401.14705
 - **Abstract**
 The availability of training data is one of the main limitations in deep learning applications for medical imaging. Data augmentation is a popular approach to overcome this problem. A new approach is a Machine Learning based augmentation, in particular usage of Generative Adversarial Networks (GAN). In this case, GANs generate images similar to the original dataset so that the overall training data amount is bigger, which leads to better performance of trained networks. A GAN model consists of two networks, a generator and a discriminator interconnected in a feedback loop which creates a competitive environment. This work is a continuation of the previous research where we trained StyleGAN2-ADA by Nvidia on the limited COVID-19 chest X-ray image dataset. In this paper, we study the dependence of the GAN-based augmentation performance on dataset size with a focus on small samples. Two datasets are considered, one with 1000 images per class (4000 images in total) and the second with 500 images per class (2000 images in total). We train StyleGAN2-ADA with both sets and then, after validating the quality of generated images, we use trained GANs as one of the augmentations approaches in multi-class classification problems. We compare the quality of the GAN-based augmentation approach to two different approaches (classical augmentation and no augmentation at all) by employing transfer learning-based classification of COVID-19 chest X-ray images. The results are quantified using different classification quality metrics and compared to the results from the literature. The GAN-based augmentation approach is found to be comparable with classical augmentation in the case of medium and large datasets but underperforms in the case of smaller datasets. The correlation between the size of the original dataset and the quality of classification is visible independently from the augmentation approach.
## Keyword: x-ray
There is no result 
### Machine learning-based analysis of glioma tissue sections: a review
 - **Authors:** Jan-Philipp Redlich, Friedrich Feuerhake, Joachim Weis, Nadine S. Schaadt, Sarah Teuber-Hanselmann, Christoph Buck, Sabine Luttmann, Andrea Eberle, Stefan Nikolin, Arno Appenzeller, Andreas Portmann, André Homeyer
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.15022
 - **Pdf link:** https://arxiv.org/pdf/2401.15022
 - **Abstract**
 In recent years, the diagnosis of gliomas has become increasingly complex. Histological assessment of glioma tissue using modern machine learning techniques offers new opportunities to support diagnosis and outcome prediction. To give an overview of the current state of research, this review examines 70 publicly available research studies on machine learning-based analysis of stained human glioma tissue sections, covering the diagnostic tasks of subtyping (16/70), grading (23/70), molecular marker prediction (13/70), and survival prediction (27/70). All studies were reviewed with regard to methodological aspects as well as clinical applicability. It was found that the focus of current research is the assessment of hematoxylin and eosin-stained tissue sections of adult-type diffuse gliomas. The majority of studies (49/70) are based on the publicly available glioblastoma and low-grade glioma datasets from The Cancer Genome Atlas (TCGA) and only a few studies employed other datasets in isolation (10/70) or in addition to the TCGA datasets (11/70). Current approaches mostly rely on convolutional neural networks (53/70) for analyzing tissue at 20x magnification (30/70). A new field of research is the integration of clinical data, omics data, or magnetic resonance imaging (27/70). So far, machine learning-based methods have achieved promising results, but are not yet used in real clinical settings. Future work should focus on the independent validation of methods on larger, multi-site datasets with high-quality and up-to-date clinical and molecular pathology annotations to demonstrate routine applicability.
### Revisiting Active Learning in the Era of Vision Foundation Models
 - **Authors:** Sanket Rajan Gupte, Josiah Aklilu, Jeffrey J. Nirschl, Serena Yeung-Levy
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.14555
 - **Pdf link:** https://arxiv.org/pdf/2401.14555
 - **Abstract**
 Foundation vision or vision-language models are trained on large unlabeled or noisy data and learn robust representations that can achieve impressive zero- or few-shot performance on diverse tasks. Given these properties, they are a natural fit for active learning (AL), which aims to maximize labeling efficiency, but the full potential of foundation models has not been explored in the context of AL, specifically in the low-budget regime. In this work, we evaluate how foundation models influence three critical components of effective AL, namely, 1) initial labeled pool selection, 2) ensuring diverse sampling, and 3) the trade-off between representative and uncertainty sampling. We systematically study how the robust representations of foundation models (DINOv2, OpenCLIP) challenge existing findings in active learning. Our observations inform the principled construction of a new simple and elegant AL strategy that balances uncertainty estimated via dropout with sample diversity. We extensively test our strategy on many challenging image classification benchmarks, including natural images as well as out-of-domain biomedical images that are relatively understudied in the AL literature. Source code will be made available.
## Keyword: radiology
There is no result 
## Keyword: radiography
There is no result 
### Fuzzy Logic-Based System for Brain Tumour Detection and Classification
 - **Authors:** NVSL Narasimham, Keshav Kumar K
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2401.14414
 - **Pdf link:** https://arxiv.org/pdf/2401.14414
 - **Abstract**
 Brain Tumours (BT) are extremely dangerous and difficult to treat. Currently, doctors must manually examine images and manually mark out tumour regions to diagnose BT; this process is time-consuming and error-prone. In recent times, experts have proposed automating approaches for detecting BT at an early stage. The poor accuracy and highly incorrect prediction results of these methods caused them to start the research. In this study, we suggest a fuzzy logic-based system for categorising BT. This study used a dataset of 253 Magnetic Resonance Imaging (MRI) brain images that included tumour and healthy images. The images were first pre-processed. After that, we pull out features like tumour size and the image's global threshold value. The watershed and region-growing approach is used to calculate the tumour size. After that, the fuzzy system receives the two features as input. Accuracy, F1-score, precision, and recall are used to assess the results of the fuzzy by employing both size determination approaches. With the size input variable discovered by the region growth method and global threshold values, the fuzzy system outperforms the watershed method. The significance of this research lies in its potential to revolutionize brain tumour diagnosis by offering a more accurate and efficient automated classification system. By reducing human intervention and providing reliable results, this approach could assist medical professionals in making timely and precise decisions, leading to improved patient outcomes and potentially saving lives. The advancement of such automated techniques has the potential to pave the way for enhanced medical imaging analysis and, ultimately, better management of brain tumour cases.
## Keyword: chexpert
There is no result
